ci/scripts/clone-build_ci.sh

```diff
@@ -72,16 +72,17 @@ cd sorc || exit 1
 set +e
 ./checkout.sh -c -g -u >> log.checkout 2>&1
 checkout_status=$?
+DATE=$(date +'%D %r')
 if [[ ${checkout_status} != 0 ]]; then
   {
     echo "Checkout: *** FAILED ***"
-    echo "Checkout: Failed at $(date)" || true
+    echo "Checkout: Failed at ${DATE}" || true
```

Comment:
Minor quib here: `DATE` here will be the `date` when `DATE` was set and not the actual date "now".

Reply:
It's set on the line above and helps with shellnorms, I should take out the || true now.

Reply:
is in the line above and is set in to places on the condition a billionth of second later (this was to appease shell norms).